src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -109,11 +109,10 @@ function renderTooltip(description: string) {
   );
 }
 
-const VISUALIZE_EMBEDDABLE_TYPE = 'visualization';
 type VisualizeEmbeddable = any;
```

Comment:
I know this wasn't added in this PR, but I'm not a huge fan of using `any` like this. Now that this `description` is added to both lens and visualize, I wonder if it would be worth adding an optional `getDescription` function to `IEmbeddable`.

If not, at least the `any` could be removed with something like:

``` typescript
type EmbeddableWithDescription = IEmbeddable & { getDescription: () => string };

function getViewDescription(embeddable: IEmbeddable) {
  if ('getDescription' in embeddable) {
    const description = (embeddable as EmbeddableWithDescription).getDescription();
    if (description) {
      return description;
    }
  }
  return '';
}
```